CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# `Helix-editor`
+### First off, Thanks for contributing! Please read the following: 
+
+Contributors are very welcome! **No contribution is too small and all contributions are valued.**
+
+Some suggestions to get started:
+
+- You can look at the [good first issue](https://github.com/helix-editor/helix/labels/E-easy) label on the issue tracker.
+- Help with packaging on various distributions needed!
+- To use print debugging to the `~/.cache/helix/helix.log` file, you must:
+  * Print using `log::info!`, `warn!`, or `error!`. (`log::info!("helix!")`)
+  * Pass the appropriate verbosity level option for the desired log level. (`hx -v <file>` for info, more `v`s for higher severity inclusive)
+- If your preferred language is missing, integrating a tree-sitter grammar for
+    it and defining syntax highlight queries for it is straight forward and
+    doesn't require much knowledge of the internals.
+
+We provide an [architecture.md](./docs/architecture.md) that should give you
+a good overview of the internals.
+
+## Before contributing:
+- Before submitting an issue, please make sure you're using the latest version of Helix, and search both closed and open issues to check that a similar issue hasn't already been submitted.
+- If you have questions about Helix's behavior or usage, please do not file an issue. Instead, ask your questions in [discussions](https://github.com/helix-editor/helix/discussions) or the [Matrix space](https://matrix.to/#/#helix-community:matrix.org).
```

Comment:
Same with this section, contributing guidelines are usually about the code workflow & PR submission. For submitting issues, bug reports and questions I find the "Getting help" section of the main README a lot more suitable

Reply:
Okay.. Well, after all of the proposed changes, it ends up with what you have in the README. So, I don't think it changes anything. Should I close this PR?

Reply:
Yeah sorry... let's drop it for now